aQute.libg/src/aQute/lib/io/IO.java

```diff
@@ -622,9 +622,6 @@ public static PrintWriter writer(OutputStream out) throws IOException {
 		return writer(out, "UTF-8");
 	}
 
-	/**
-	 * Reflective way to create a link. This assumes Java 7+
-	 */
 	public static boolean createSymbolicLink(File link, File target) throws Exception {
 		try {
 			Files.createSymbolicLink(link.toPath(), target.toPath());
```

Comment:
I think this code needs to handle the case where the link already exists and is pointing to the requested target. In that case, return true. (Alternately, if could test if link is a symbolic link already, and it so, delete it before creating a new link that points to the target file.)

I think the current code will catch an exception and return false which means createSymbolicLinkOrCopy will then copy.

This change is needed since we cannot always delete the link before calling createSymbolicLinkOrCopy since it may be a copy which is unchanged from the target.